### ISSUE TITLE
Populate R2D2 with bgc observations

### ISSUE THREAD

**Issue by guillaumevernieres** [NOAA-EMC/godas#374]
# Description
Populate the NOAA-EMC R2D2 observation database with:
-  PACE synthetic observations https://oceandata.sci.gsfc.nasa.gov/directdataaccess/Level-1C/PACE_SPEXONE/
- .. @liuxiao37k to fill in

**Comment by guillaumevernieres** [comment]
@liuxiao37k , @mjhossen , were the synthetic PACE obs moved to R2D2?

**Comment by mjhossen** [comment]
> @liuxiao37k , @mjhossen , were the synthetic PACE obs moved to R2D2?

They are ready to move. I will request @ShastriPaturi to move them.

**Comment by mjhossen** [comment]
@ShastriPaturi Would you please move the pace obs to r2d2 database.  Both the 10min and superobs are located in the folder: /work2/noaa/ng-godas/jhossen/pace_out/

**Comment by liuxiao37k** [comment]
@mjhossen Have the PACE obs been reprocessed since November 10 when this issue was identified? see the first highlighted line [here](https://docs.google.com/document/d/1hRyRN9EPulKhIYYGml60g7Jp8MMDFxXi/edit), also see Slack discussion [here](https://noaa-emc.slack.com/archives/C03AH0MRAHW/p1669732452904679). Also, are the processed PACE obs the new ones that Cecile shared back in October? thank you!
@ShastriPaturi

**Comment by mjhossen** [comment]
@liuxiao37k Yes, I have reprocessed the new PACE obs data with the issue that we discussed.

**Comment by mjhossen** [comment]
@liuxiao37k Yes, I have reprocessed the data with the issue that we figured out. I process new ones (Level 1B).
